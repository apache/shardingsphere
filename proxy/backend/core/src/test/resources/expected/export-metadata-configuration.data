eyJtZXRhX2RhdGEiOnsiZGF0YWJhc2VzIjp7Im5vcm1hbF9kYiI6ImRhdGFiYXNlTmFtZTogbm9ybWFsX2RiXG5kYXRhU291cmNlczpcbiAgZHNfMDpcbiAgICBwYXNzd29yZDogXG4gICAgdXJsOiBqZGJjOm9wZW5nYXVzczovLzEyNy4wLjAuMTo1NDMyL2RlbW9fZHNfMFxuICAgIHVzZXJuYW1lOiByb290XG4gICAgbWluUG9vbFNpemU6IDFcbiAgICBtYXhQb29sU2l6ZTogNTBcbiAgZHNfMTpcbiAgICBwYXNzd29yZDogXG4gICAgdXJsOiBqZGJjOm9wZW5nYXVzczovLzEyNy4wLjAuMTo1NDMyL2RlbW9fZHNfMVxuICAgIHVzZXJuYW1lOiByb290XG4gICAgbWluUG9vbFNpemU6IDFcbiAgICBtYXhQb29sU2l6ZTogNTBcbiJ9LCJwcm9wcyI6InByb3BzOlxuICBzcWwtc2hvdzogdHJ1ZVxuIiwicnVsZXMiOiJydWxlczpcbi0gIUFVVEhPUklUWVxuICBwcml2aWxlZ2U6XG4gICAgdHlwZTogQUxMX1BFUk1JVFRFRFxuICB1c2VyczpcbiAgLSBhdXRoZW50aWNhdGlvbk1ldGhvZE5hbWU6ICcnXG4gICAgcGFzc3dvcmQ6IHJvb3RcbiAgICB1c2VyOiByb290QCVcbi0gIUdMT0JBTF9DTE9DS1xuICBlbmFibGVkOiBmYWxzZVxuICBwcm92aWRlcjogbG9jYWxcbiAgdHlwZTogVFNPXG4ifX0=
