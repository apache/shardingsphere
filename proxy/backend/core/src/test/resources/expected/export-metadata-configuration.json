{"meta_data":{"databases":{"normal_db":"databaseName: normal_db\ndataSources:\n  ds_0:\n    password: \n    url: jdbc:opengauss://127.0.0.1:5432/demo_ds_0\n    username: root\n    minPoolSize: 1\n    maxPoolSize: 50\n  ds_1:\n    password: \n    url: jdbc:opengauss://127.0.0.1:5432/demo_ds_1\n    username: root\n    minPoolSize: 1\n    maxPoolSize: 50\n"},"props":"props:\n  sql-show: true\n","rules":"rules:\n- !AUTHORITY\n  privilege:\n    type: ALL_PERMITTED\n  users:\n  - authenticationMethodName: ''\n    password: root\n    user: root@%\n- !GLOBAL_CLOCK\n  enabled: false\n  provider: local\n  type: TSO\n"}}
